planning/obstacle_cruise_planner/config/obstacle_cruise_planner.param.yaml

```diff
@@ -217,3 +217,20 @@
       lpf_gain_slow_down_vel: 0.99 # low-pass filter gain for slow down velocity
       lpf_gain_lat_dist: 0.999 # low-pass filter gain for lateral distance from obstacle to ego's path
       lpf_gain_dist_to_slow_down: 0.7 # low-pass filter gain for distance to slow down start
+    stop:
+      type_specified_params:
+        use_type_specified_params: # If true, the node try to read the following type specified values
+          unknown: true
+          car: false
+          truck: false
+          bus: false
+          trailer: false
+          motorcycle: false
+          bicycle: false
+          pedestrian: false
+        limit_min_acc: # overwrite the deceleration limit, in usually, common_param.yaml/limit.min_acc is referred.
+          unknown: -1.0
+        limit_max_dist:
+          unknown: 50.0
+        abandon_to_stop: # If true, the planner gives up to stop when it cannot avoid a collision while maintaining the deceleration limit.
+          unknown: false
```

Comment:
Is it possible to list down parameters depending on the object type as parameters of `slow_down` ?